Sync Properties Update Revision Block and Create PDF/Modules/Sample.SyncFilePropsAndUpdRevBlockModule.psm1

```diff
@@ -843,4 +844,15 @@ function UpdateRevisionBlock {
 		throw "Failed job 'Update Revision Block'" #Failed because of issue that occured in the job
 	}
 	Write-Host "End 'Update Revision Block' of file '$($fileForUpdRevJob.Name)'"
+
+	# $versionIds = @($file.Id)
+	# $fileVersions = [Connectivity.Services.Document.DocServices]::Instance.GetFileVersionsByIDs($vaultConnection, $versionIds)
+	# $multiSideProvider = new-object Connectivity.Explorer.JobHandlerUpdateRevisionBlock.URBJobHandlerMultiSiteSyncProvider -ArgumentList $true
+
+	# $updateRevisionBlock = new-object Connectivity.Explorer.Document.ViewModel.UpdateRevisionBlock -ArgumentList $fileVersions, $false
+	# $updateRevisionBlock.MultiSiteSyncProvider = $multiSideProvider
+	# $updateRevisionBlock.CheckInComment = "Revision table updated by coolOrange UpdateRevisionTable"
+	# $updateRevisionBlock.RefreshRevisionBlock()
+
 }
+
```

Comment:
not sure why this is commented. is this still in progress?

Reply:
yes, I currently get a null refrence error in line 855